eq-author-api/schema/resolvers/base.js

```diff
@@ -514,6 +514,8 @@ const Resolvers = {
       }
       return "Read";
     },
+    totalErrorCount: (questionnaire, args, ctx) =>
+      ctx.validationErrorInfo ? ctx.validationErrorInfo.totalCount : 0,
```

Comment:
Why is this check necessary?

Reply:
The 3 tests under `duplicate a questionnaire` in `eq-author-api/schema/tests/duplication.test.js` was failing without it. Giving totalCount is undefined.

Reply:
I guess this is an issue with duplication not validating the questionnaire